r_markdown/ef_qaqc_template.Rmd

```diff
@@ -0,0 +1,245 @@
+---
+title: "EF Template"
+author: "Kelsey Bilsback"
+date: "02/07/2017"
+output: html_document
+---
+
+```{r, echo=FALSE, warning=FALSE, message=FALSE, results='markup', cache=FALSE}
+  library(tidyverse)
+  library(forcats)
+```
+
+```{r, echo=FALSE, warning=FALSE, message=FALSE, results='markup', cache=FALSE}
+  source("../r_scripts/R_tidy.R")
+```
+
+## Load data
+
+```{r, echo=FALSE, warning=FALSE, message=FALSE, results='markup', cache=FALSE}
+  load("../r_files/emissions.Rda")  # load emissions data
+  load("../r_files/samples.Rda")  # load sample meta data
+  load("../r_files/fuel_burnt.Rda")  # fuel use meta data
+```
+
+* Output emissions column names 
+
+```{r, echo=FALSE, warning=FALSE, message=FALSE, results='markup', cache=FALSE}
+  colnames(emissions)
+```
+
+## Select pollutant of interest
+
+```{r, echo=TRUE, warning=FALSE, message=FALSE, results='markup', cache=FALSE}
+  pol_p <- dplyr::filter(emissions,  pol == "voc_benzene")
+  pol_name <- "benzene"
+```
+
+## QAQC
+
+* Add QAQC flag
+* Filter out bad tests
+
+```{r, echo=TRUE, warning=FALSE, message=FALSE, results='markup', cache=FALSE}
+  pol_p <- dplyr::filter(pol_p, qc != "bad")
+```
+
+## Calculate emissions factor
+
+```{r, echo=TRUE, warning=FALSE, message=FALSE, results='markup', cache=FALSE}
+  pol_p <- dplyr::mutate(pol_p, mass_ef = 
+                           (mass_emitted * 1e-3) / (mass_fuel*1e-3)) %>% # mg/kg
```

Comment:
you need to join with the fuel burnt df before this will work e.g. mass_fuel is not in df pol_p yet etc...